src/DotNetProjectFile.Analyzers/Text/Levenshtein.cs

```diff
@@ -0,0 +1,82 @@
+namespace DotNetProjectFile.Text;
+
+/// <summary>
+/// Measures the difference between two strings.
+/// Uses the Levenshtein string difference algorithm.
+///
+/// See: https://en.wikipedia.org/wiki/Levenshtein_distance.
+/// </summary>
+/// <remarks>
+/// Copyright: Dan Hartley, https://github.com/DanHarltey/Fastenshtein.
```

Comment:
MIT license requires a reproduction of the license text with any reproduction of the code, so I would also include that here in the remarks.